sources/packages/backend/libs/services/src/students-assessments/assessment-sequential-processing.service.ts

```diff
@@ -343,4 +385,139 @@ export class AssessmentSequentialProcessingService {
       options?.alternativeReferenceDate,
     );
   }
+
+  /**
+   * Get SFAS application awards totals.
+   * @param assessment current assessment being processed.
+   * @param firstAssessmentDate date of the first assessment date of the current application.
+   * @returns SFAS application awards totals.
+   */
+  async getProgramYearSFASAwardsTotals(
+    assessment: StudentAssessment,
```

Comment:
I would recommend to use the individual properties here, to have the signature of method more clear.

like this.

```ts
  async getProgramYearSFASAwardsTotals(
    lastName: string,
    birthDate: string,
    sin: string,
    programYearStartDate: string,
    firstAssessmentDate: string,
  ): Promise<AwardTotal[]>
```